external/libcxx-test/libcxx_tc_common.h

```diff
@@ -0,0 +1,41 @@
+/****************************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __EXTERNAL_LIBCXX_TEST_LIBCXX_TC_COMMON_H
+#define __EXTERNAL_LIBCXX_TEST_LIBCXX_TC_COMMON_H
+#include "tc_common.h"
+
+/* Custom TC macros */
+
+#define TC_ASSERT_EXPR(expr)	\
```

Comment:
This is same as ```TC_ASSERT```. Why do you define this even though you include "tc_common.h"?

Reply:
It is not same as TC_ASSERT.
It has differed in two ways.
1. Expects only one argument (expression) same as assert. So it is to replace to original assert calls.
2. It returns return value, Because this return value will be used to check TC_ASSERT_FUNC.